src/Badge.js

```diff
@@ -0,0 +1,88 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { space } from 'styled-system'
+import { theme } from '../'
+
+const type = props => {
+  switch(props.type) {
+    case 'success':
+      return {
+        backgroundColor: theme.colors.green,
+        color: theme.colors.white
+      }
+    case 'success-light':
+      return {
+        backgroundColor: theme.colors.green0,
+        color: theme.colors.black
+      }
+    case 'warning':
+      return {
+        backgroundColor: theme.colors.red,
+        color: theme.colors.white
+      }
+    case 'warning-light':
+      return {
+        backgroundColor: theme.colors.red0,
+        color: theme.colors.black
+      }
+    case 'attention':
+      return {
+        backgroundColor: theme.colors.orange,
+        color: theme.colors.white
+      }
+    case 'attention-light':
+      return {
+        backgroundColor: theme.colors.orange0,
+        color: theme.colors.black
+      }
+    case 'information':
+      return {
+        backgroundColor: theme.colors.blue,
+        color: theme.colors.white
+      }
+    case 'information-light':
+      return {
+        backgroundColor: theme.colors.blue0,
+        color: theme.colors.black
+      }
+    default:
```

Comment:
What about mapping the type prop to an object key that points to a set of background color and color instead?

E.g.
```
const badgeColors = {
  'success': {
    backgroundColor: theme.colors.green,
    color: theme.colors.white
  }
}
```

Then in the function
```
return badgeColors[props.type]
```

Or even making each badge color a styled component that extends Badge, and then map them to keys after that. It will save a tiny bit of computing time.